app/src/main/java/edu/uc/willi6jd/mycaloriecounter/RecipeActivity.kt

```diff
@@ -20,29 +26,22 @@ class RecipeActivity : AppCompatActivity() {
         btnCreate.setOnClickListener {
 
             // Get input fields (by ID) store them to recipe item variables
-            var itemName = findViewById<EditText>(R.id.recipeName); //Name of recipe: input text field
-            var itemCount = findViewById<EditText>(R.id.recipeItemCount); //Number of food items in recipe: integer
-            var itemDescription = findViewById<EditText>(R.id.recipeDescription); //Description of recipe: input text field, string
+            var itemName = findViewById<EditText>(R.id.recipeName).toString(); //Name of recipe: input text field
+            var itemCount = findViewById<EditText>(R.id.recipeItemCount).toInt(); //Number of food items in recipe: integer
+            var itemDescription = findViewById<EditText>(R.id.recipeDescription).toString(); //Description of recipe: input text field, string
```

Comment:
Ok, I see, yes... this is similar to what I mentioned above.  Invoke toString() to get the String value to pass to the function.